extensions/wikia/PortabilityDashboard/SpecialPortabilityDashboardController.php

```diff
@@ -22,18 +22,19 @@ public function index() {
 		$wikiUrlParam = $this->getVal( static::WIKI_URL_QS_PARAM, '' );
 		$isLangFilterSet = !empty( $langFilter );
 		$isWikiUrlParamSet = !empty( $wikiUrlParam );
-		$noResultsInfoMessage = wfMessage(
-			$isLangFilterSet ? 'portability-dashboard-no-results-for-lang-info' : 'portability-dashboard-no-results-for-url-info',
-			PortabilityDashboardModel::WIKIS_LIMIT
-		)->text();
+		$noResultsInfoMessageKey = 'portability-dashboard-no-results-for-url-info';
```

Comment:
In this case probably the 'portability-dashboard-no-results-for-url-info' key is misleading